### DISCUSSION TITLE
Checkbox to assign OOB on IP address creation

### DISCUSSION THREAD

**Discussion by llamafilm** [netbox-community/netbox#14594] [Ideas]
Since Netbox now has the ability to assign OOB IP address to a device (thanks to #13013), I think it would be helpful to add a checkbox for this when adding a new IP address, in the same way you can assign the Primary IP.  I propose adding the checkbox right below this one, labeled "Make this the Out-of-band IP for the device"
<img width="736" alt="Screenshot 2023-12-22 at 10 09 24 AM" src="https://github.com/netbox-community/netbox/assets/3991046/00192e0d-7d0c-4ab9-a21d-f4bb6a3c9e18">

As it is now, I have to first create the IP address and assign to the interface, then go to the device edit page and set it as Out-of-band IP here:
<img width="790" alt="Screenshot 2023-12-22 at 10 09 41 AM" src="https://github.com/netbox-community/netbox/assets/3991046/c97d0da5-5755-48e7-8395-007db83cad68">

**Comment by novaclark** [comment]
This would be a huge time-saver for us. I'd love to be able to mass-edit a bunch of IP addresses and mark them as OOB rather than individually editing a bunch of devices.
